### DISCUSSION TITLE
How to debug formatter (prettier) not working randomly?

### DISCUSSION THREAD

**Discussion by tarikhamilton** [LunarVim/LunarVim#3633] [Q&A]
I'm using LunarVim with a TypeScript React project and prettier. Formatting will randomly stop working. I can open up the same exact file in VS Code and it'll format it just fine.

My lvim's config.lua for formatting is super simple.

```
lvim.format_on_save = true

local formatters = require "lvim.lsp.null-ls.formatters"
formatters.setup {
  { command = "prettier" },
}
```

I've run `:HealthCheck`, which says my `prettier` command works. Everything else is OK too.

Since I can't find any issues, I'm going to assume it's me, but I don't really know what else I can check to figure out what's wrong or how this keeps happening.

**Comment by LostInTheLogs** [comment]
Maybe enable nullls debugging and check the null-ls log which is in .config/nvim (and not lvim)
